### PR TITLE
Adding an ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by yohanboniface**
First suggestion. Thoughts?

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+- [ ] I'm reporting a bug, not asking for help
+- [ ] I've looked at the [documentation](http://leafletjs.com/reference-1.0.0.html) to make sure this is not a documented behaviour
```
Comment (IvanSanchez) [inline ISSUE_TEMPLATE.md]: Typo: «not documented behaviour». Maybe it's clearer if the wording is «to make sure the behaviour is documented and expected»

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] I'm sure this is a Leaflet code issue, not an issue with my own code
```
Comment (IvanSanchez) [inline ISSUE_TEMPLATE.md]: I'd add a line about ionic, cordova, react, ractive, vue and all the other JS frameworks we don't (really) care about.

**Comment by IvanSanchez** [comment]
I'd also add a couple of lines about specifying the browser for browser-specific bugs (or the more generic word "platform" to include phones and such), and also (very important) which version the bug is reproducible in.

**Comment by yohanboniface** [comment]
> I'd also add a couple of lines about specifying the browser for browser-specific bugs (or the more generic word "platform" to include phones and such),

Right! :)

> and also (very important) which version the bug is reproducible in.

Damn right! :)

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (yohanboniface) [inline ISSUE_TEMPLATE.md]: Changed :)

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (yohanboniface) [inline ISSUE_TEMPLATE.md]: Added :)

**Comment by mourner** [comment]
I like it, although I would maybe simplify some statements and formatting to make the template shorter (longer templates are less likely to be adhered to).

**Comment by yohanboniface** [comment]
I guess those two sentences can be made shorter:

> - [ ] I've looked at the [documentation](http://leafletjs.com/reference-1.0.0.html) to make sure the behaviour is documented and expected
>- [ ] I'm sure this is a Leaflet code issue, not an issue with my own code nor with the framework I'm using (Cordova, Ionic, Angular, React…)

To you have any other idea in mind?

**Comment by danzel** [comment]
The language on this bit could be improved I think:
> this example do not imply any third party code

I'm not sure what it is meant to mean.

**Comment by perliedman** [comment]
@danzel a changed "imply" to "rely on", which at least for me makes more sense!

**Comment by yohanboniface** [comment]
Great, thanks Per :)

**Comment by mourner** [comment]
@yohanboniface maybe you meant "does not", not "do not"?

**Comment by yohanboniface** [comment]
> @yohanboniface maybe you meant "does not", not "do not"?

Erf, fixed :p

**Pull request merged**